docs/v2/manual.md

```diff
@@ -649,7 +649,7 @@ func main() {
 
   app := &cli.App{
     Action: func(c *cli.Context) error {
-      fmt.Println("yaml ist rad")
+      fmt.Println("--test value.*default: 0")
```

Comment:
apologies for this part. I am not really sure whats happening her I just tried a few things locally to get the tests to pass. Seems like they fail on master rather than any change I have made.

Please let me know if you would like me to revert this change.